tsk/auto/is_image_supported.cpp

```diff
@@ -44,6 +46,47 @@ bool TskIsImageSupported::isImageEncrypted()
     return m_wasEncryptionFound;
 }
 
+/**
+* Idea is to try to give the user a simple error message explaining the most likely 
+* reason the image is not supported
+*/
+std::string TskIsImageSupported::getSingleLineErrorMessage() {
+    // If we have this, we are very confident we have a BitLocker-protected partition
+    // and that we either need a password or that the one given was incorrect.
+    if (m_bitlockerError) {
+        if (strnlen(m_bitlockerDesc, 1024) > 0) {
+            return std::string(m_bitlockerDesc);
+        }
+        return "BitLocker error";
```

Comment:
Do we ever expect this generic "Bitlocker error" to be returned or is this just a safety message?